conanfile.py

```diff
@@ -81,9 +70,6 @@ def requirements(self):
                 self.requires(req)
```

Comment:
_GH won't allow me to add a comment above this line._

Maybe rename this option to `enable_testing` like we do in CuraEngine

Did you also test this functionality?
```
    def build_requirements(self):
        if self.options.tests:
            for req in self._um_data()["requirements_testing"]:
                self.tool_requires(req, force_host_context=True)
```

Reply:
https://github.com/Ultimaker/libnest2d/pull/8/commits